sdk/service/sdk.go

```diff
@@ -44,6 +47,9 @@ func (s *SDK) Create(req *api.CreateServiceRequest, accountName, accountPassword
 	msg := newMsgCreateService(s.cdc, req, owner)
 	tx, err := s.client.BuildAndBroadcastMsg(msg, accountName, accountPassword, accNumber, accSeq)
 	if err != nil {
+		if err == mempool.ErrTxInCache {
```

Comment:
I don't like to check this error. This error could never happen if the `accSeq` changes.
What about a pre-check on that calculate the hash and check if it already exists?

Reply:
>This error could never happen if the accSeq changes.

Ąre you sure?

I've made a test and error is still the same if `accSeq` changes.

 I also think that we should leave that check to network, as the network will validate that for us better (for example it handles the race condition when one service is created at the same time more than once).

So that check won't give us anserw if the service exists or not. Engine have to still handle that error case.